.github/workflows/version-bump.yml

```diff
@@ -52,6 +21,6 @@ jobs:
           args: ${{ secrets.CRATES_IO_TOKEN }}
       - name: publish
         run: |
-          if [ -f publishMe ]; then
-            cargo publish
-          fi
+          echo 'Uploading to crates.io'
```

Comment:
Uh, I'm confused now, this _always_ pushes to crates.io?

Anyway, you probably want to rename this file? It seems there was some naming confusion here.

Reply:
This workflow is invoked right now only manually, so yes, if you invoke it manually, it will publish every time it will be called.

Reply:
Ok, I did some clean up. See 4339e93. Workflow should be like this:
1. On PR creation to main  -> prepare-version-bump.yml 
2. Manually (for now) -> version-bump.yml
3. On (2) -> push-foundation-repo.yml.

Reply:
Yeah, that seems good now. I still have some gripes with naming – maybe call this one something like "Publish to crates.io" rather than "Autobump version"? Similar for file names, what is now called "prepare-..." should just be "version-bump" and this one "publish" or something.